core/common/modules/connect/module.php

```diff
@@ -92,6 +92,17 @@ public function init() {
 		foreach ( $this->registered_apps as $slug => $class ) {
 			$this->apps[ $slug ] = new $class();
 		}
+
+		add_filter( 'elementor/editor/localize_settings', [ $this, 'localize_settings' ] );
+	}
+
+	public function localize_settings( $settings ) {
+		return array_replace_recursive( $settings, [
+			'i18n' => [
+				'connect_error' => __( 'Unable to connect', 'elementor' ),
+				'connected_successfully' => __( 'Connected successfully', 'elementor' ),
+			],
+		] );
```

Comment:
You don't need to return back the filter and the content. Just keep an empty method and add BC doc.